app/src/main/java/com/pvp/app/ui/screen/friends/FriendsScreen.kt

```diff
@@ -0,0 +1,435 @@
+                        ) {
+                            Text(
+                                "Received",
+                                style = MaterialTheme.typography.titleMedium,
+                                fontWeight = if (selectedTab.intValue == 0) FontWeight.Bold else FontWeight.Normal
+                            )
+                        }
+                        Tab(
+                            selected = selectedTab.intValue == 1,
+                            onClick = { selectedTab.intValue = 1 }
+                        ) {
+                            Text(
+                                "Sent",
+                                style = MaterialTheme.typography.titleMedium,
+                                fontWeight = if (selectedTab.intValue == 1) FontWeight.Bold else FontWeight.Normal
+                            )
+                        }
+                    }
+
+                    Spacer(modifier = Modifier.height(6.dp))
+
+                    when (selectedTab.intValue) {
+                        0 -> {
+                            RequestList(
+                                requests = receivedRequests,
+                                requestTitle = "received",
+                                acceptAction = { request -> model.acceptFriendRequest(request) },
+                                denyAction = { request -> model.denyFriendRequest(request) }
+                            )
+                        }
+
+                        1 -> {
+                            RequestList(
+                                requests = sentRequests,
+                                requestTitle = "sent",
+                                acceptAction = { },
+                                denyAction = { request -> model.cancelSentRequest(request) }
+                            )
+                        }
+                    }
+                }
+            },
+        )
+    }
+}
+
+@Composable
+private fun TopRowWithButtons(
+    showAddFriend: MutableState<Boolean>,
+    showRequests: MutableState<Boolean>
+) {
+    Row(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(
+                start = 16.dp,
+                end = 16.dp,
+                top = 4.dp
+            ),
+        horizontalArrangement = Arrangement.SpaceBetween,
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        RowButton(
+            onClick = { showAddFriend.value = true },
+            modifier = Modifier.size(
+                width = 45.dp,
+                height = 35.dp
+            ),
+            content = ButtonContent.Icon(Icons.Outlined.GroupAdd)
+        )
+
+        RowButton(
+            onClick = { showRequests.value = true },
+            modifier = Modifier.size(
+                width = 160.dp,
+                height = 35.dp
+            ),
+            content = ButtonContent.Text("Requests")
+        )
+
+        RowButton(
+            onClick = { /* TODO */ },
```

Comment:
What is this for? Comment doesn't really help to understand what has to be implemented.